osbs/build/user_params.py

```diff
@@ -280,6 +284,15 @@ def set_params(self,
         self.signing_intent.value = signing_intent
         self.compose_ids.value = compose_ids or []
 
+        try:
+            self.orchestrator_deadline.value = string.atoi(orchestrator_deadline)
```

Comment:
`string.atoi` is not available in python3 (and is deprecated in python2). You may want to use the `int()` builtin here. https://docs.python.org/2/library/string.html#string.atoi

Reply:
Fixed.